docs/networking/deep-dive.md

```diff
@@ -0,0 +1,112 @@
+
+The Harvester network is divided into three layers, including:
+
+- KubeVirt networking layer
+
+- Harvester networking layer
+
+- external networking layer
+
+## Kubevirt Networking
+
+The basic idea of KubeVirt is to put libvirt into the Kubernetes pod. The KubeVirt network is responsible for forwarding traffic between the pod and VM. 
+Please refer to the [KubeVirt official document](https://kubevirt.io/2018/KubeVirt-Network-Deep-Dive.html) for more details.
+
+## Harvester Networking
+
+Harvester networking is designed to forward the traffic from pods to the host network. It implements a management network, VLAN networks and untagged networks. We can refer to the last two networks as **bridge networks**, because bridge plays a vital role in their implementation.
+
+### Bridge Network
+
+We leverage [multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni) and [bridge CNI](https://www.cni.dev/plugins/current/main/bridge/) to implement the bridge network.
+
+1. Multus CNI is a Container Network Interface (CNI) plugin for Kubernetes that can attach multiple network interfaces to a pod. Its capability allows a VM to have one NIC for the management network and multiple NICs for the bridge network.
+
+2. Using the bridge CNI, the VM pod will be plugged into the L2 bridge specified in the Network Attachment Definition config.
+
+    ```json
+    # Example 1
+    {
+        "cniVersion": "0.3.1",
+        "name": "vlan100",
+        "type": "bridge",
+        "bridge": "mgmt-br",
+        "promiscMode": true,
+        "vlan": 100,
+    }
+    ```
+
+    ```json
+    # Example 2
+    {
+        "cniVersion": "0.3.1",
+        "name": "untagged-network",
+        "type": "bridge",
+        "bridge": "oob-br",
+        "promiscMode": true,
+        "ipam": {}
+    }
+    ```
+
+    Example 1 is a typical VLAN configuration with VLAN ID 100, while Example 2 is an untagged network configuration with no VLAN ID. The VM pod configured using Example 1 will be plugged into the bridge `mgmt-br`, while the VM pod using Example 2 will be plugged into the bridge `oob-br`.
+
+3. For fault tolerance and high availability concerns, the uplink of each bridge is a bond device. For a bridge port, the uplink should allow the tagged package to pass directly.
+
+    ```shell
+    harvester-0:/home/rancher # bridge -c vlan show dev oob-bo
+    port	   vlan ids
+    oob-bo	   1 PVID Egress Untagged
+               100
+               200
+    ```
+
+    The example above shows that the bond `oob-bo` allows packages with tag 1, 100 or 200.
+
+### Management Network
+
+The management network is based on [Canal](https://projectcalico.docs.tigera.io/getting-started/kubernetes/flannel/flannel).
+
+It is worth mentioning that the Canal interface uses the bridge `mgmt-br` and the VLAN sub-interface of `mgmt-br` where the Harvester configures its node IP. This design has two benefits:
```

Comment:
is `mgmt-br` created by default and can not be deleted? when true, please highlight it here or in the previous part